ddtrace/debugging/_debugger.py

```diff
@@ -24,7 +24,7 @@
 from ddtrace._trace.tracer import Tracer
 from ddtrace.debugging._config import di_config
 from ddtrace.debugging._function.discovery import FunctionDiscovery
-from ddtrace.debugging._function.store import FullyNamedWrappedFunction
+from ddtrace.debugging._function.store import FullyNamedContextWrappedFunction
```

Comment:
is this needed for 3.13 support?

Reply:
🤔 this looks like a slight refactor I did. I don't think it is needed for 3.13 support so I'll see if I can take this out in its own PR

Reply:
https://github.com/DataDog/dd-trace-py/pull/11721